resource/research2022-prod/session_map.json

```diff
@@ -0,0 +1,14 @@
+{
+  "SessionSetMap": [
+    {
+      "HubbId": 1001,
+      "SessionTypeName": "Breakout",
+      "SessionSets": ["Session Catalog", "Session Scheduler", "My Schedule"]
+    },
+    {
+      "HubbId": 1002,
+      "SessionTypeName": "Plenary",
```

Comment:
@nelson-w I haven't seen this session type elsewhere in our solution, but it was this session type name given in the access grid - am I correct in thinking this should be a new session type?

Reply:
Yes that's correct.